docs/guide/setup/creating_ligand_networks.rst

```diff
@@ -1,6 +1,51 @@
 .. _define_ligand_network:
+.. _userguide_ligand_network:
 
 Defining the Ligand Network
 ===========================
+A ligand network is a set of small molecules connected by mappings of two ligands.
+Such a network can represent a set of drug candidates derived from molecule enumeration that should
+be ranked by free energy calculations, in order to prioritize molecule synthesis efforts.
+The ligand networks are a tool that is used to orchestrate the free energy calculations to efficiently
+compute a ligand ranking.
+It is of course possible to calculate all possible transformations defined by all possible mappings connecting all small molecules with a ''maximal network'',
+but it is much more efficient to use a network with less transformations like a ''radial network'' (also known as a star map)
+or a ''minimimal spanning network''.
 
-issue #758
\ No newline at end of file
+Any ``LigandNetwork`` generation can be generally conceptualized into three steps:
+
+* Generate the :ref:`Atom Mappings<Creating Atom Mappings>`  of all pairwise combinations of ligands
+* :ref:`Score all resulting Atom Mappings<Creating Atom Mappings>`
+* Build a :class:`.LigandNetwork` with all possible mappings directed by their scores.
+
+.. image:: img/ligand_network.png
+   :width: 90%
+   :align: center
+   :alt: Concept of a simple MST ligand network
+
+
+Generating Ligand Networks
```

Comment:
How about a section / bit on visualizing a network?

Reply:
future work

Reply:
@RiesBen if it doesn't already exist, could you please raise a relevant issue on the tracker so we don't forget about it?